### PR TITLE
updated preprocessing_directory.py to read .png and .jpg files

### PR DESCRIPTION
**Pull request by ragerdude**
# Description

Replace with description of PR

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: data_management/autopreprocess_testing/preprocess_directory.py [hunk 1]
```diff
@@ -19,12 +19,15 @@
...
 
 # Process each image and save in target directory
 for file_path in img_files:
-    file_name = os.path.basename(file_path)[:-4]
+    file_name = os.path.basename(file_path)
     img = cv2.imread(file_path)
     img = processing_pipeline(img)
-    processed_path = target_dir + file_name + "_PROCESSED.jpg"
-    cv2.imwrite(processed_path, img)
+    processed_path = target_dir + file_name
```
Comment (bofori20) [inline data_management/autopreprocess_testing/preprocess_directory.py:31]: This is really helpful for dealing with all the data we hope to add.

**Review by bofori20 — APPROVED** [review]
This is a helpful addition to the work of the precious team.

**Review by jraval — APPROVED** [review]
These changes will help organize our images in an efficient manner.  Thanks Matt!

**Review by nlittlepoole — APPROVED** [review]
:shipit: Great Work!

**Pull request merged**